pkg/settings/settings.go

```diff
@@ -132,3 +138,40 @@ func GetVDDKImage(providerSpecSettings map[string]string) string {
 
 	return vddkImage
 }
+
+func getOpenshiftConsoleOrigin() (string, error) {
+	cfg, err := rest.InClusterConfig()
+	if err != nil {
+		return "", err
+	}
+
+	dynClient, err := dynamic.NewForConfig(cfg)
+	if err != nil {
+		return "", err
+	}
+
+	routeGVR := schema.GroupVersionResource{
+		Group:    "route.openshift.io",
+		Version:  "v1",
+		Resource: "routes",
+	}
+
+	obj, err := dynClient.
+		Resource(routeGVR).
+		Namespace("openshift-console").
+		Get(context.Background(), "console", metav1.GetOptions{})
+	if err != nil {
+		return "", fmt.Errorf("failed to get console Route: %w", err)
+	}
+
+	host, found, err := unstructured.NestedString(obj.UnstructuredContent(), "spec", "host")
+	if err != nil {
+		return "", err
+	}
+
+	if !found {
+		return "", fmt.Errorf("console Route is missing spec.host")
```

Comment:
Not sure if we should fail the whole inventory when the route is missing.

Reply:
routes are OCP specific, we don't have them on K8s clusters, meaning our KIND baded CI will fail